GitUI/FormProcess.cs

```diff
@@ -155,12 +154,13 @@ private void processStart(FormStatus form)
 
                 if (!string.IsNullOrEmpty(ProcessInput))
                 {
-                    throw new NotSupportedException("No non-NULL usages of ProcessInput are currently expected.");	// Not implemented with all terminal variations, so let's postpone until there's at least one non-null case
-/*
-                    Thread.Sleep(500);
-                    Process.StandardInput.Write(ProcessInput);
-                    AddMessageLine(string.Format(":: Wrote [{0}] to process!\r\n", ProcessInput));
-*/
+                    throw new NotSupportedException("No non-NULL usages of ProcessInput are currently expected.");  // Not implemented with all terminal variations, so let's postpone until there's at least one non-null case
+
+                    /*
+                                        Thread.Sleep(500);
+                                        Process.StandardInput.Write(ProcessInput);
+                                        AddMessageLine(string.Format(":: Wrote [{0}] to process!\r\n", ProcessInput));
+                    */
```

Comment:
Not a biggie, but this is maybe a bit worse.

Reply:
I can fix this one easily...

Reply:
Again, I was commenting more on the tool than your PR.